broker/lib/cf.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import requests
+from functools import cache
+from urllib.parse import urljoin
+
+from broker.extensions import config
+
+
+class CFAPIClient:
+    _access_token: str
+    _access_token_expiration: float
+
+    def set_access_token(self):
```

Comment:
`set_<x>` reads like I should be passing a value to this method - maybe this should be `fetch_access_token`?